syntax/header.php

```diff
@@ -74,7 +74,9 @@ function render($mode, Doku_Renderer $renderer, $data) {
         if ($mode == 'xhtml') {
             switch ($data[0]) {
                 case 'open':
-                    $renderer->doc .= DOKU_LF . '<div class="qna-header">';
+                    $style = '-' . $this->getConf('style');
+                    if ($style == '-default') $style = '';

```

Comment:
This code duplication is bugging me. Consider moving it to a common base class for all syntax plugins. It will be more code, but without repetitions. In the base class you could load the style in constructor and have a protected getter for it:
```
 $renderer->doc .= DOKU_LF . '<div class="qna-header' . $this->getStyle() . '">';
```
Or even:
```
 $renderer->doc .= DOKU_LF . $this->renderDiv('qna-header');
```
In this case style would be uniformly applied to all `<div>` elements. But maybe that's taking it too far :)